colin/core/target.py

```diff
@@ -92,3 +100,58 @@ def is_compatible(target_type, check_class, severity, tags):
     return (target_type == TargetType.DOCKERFILE and isinstance(check_class, DockerfileCheck)) \
            or (target_type == TargetType.CONTAINER and isinstance(check_class, ContainerCheck)) \
            or (target_type == TargetType.CONTAINER_IMAGE and isinstance(check_class, ImageCheck))
+
+
+class ImageName(object):
```

Comment:
Would be awesome to have this in conu. At some point.

Reply:
Yes, I will look at it with [this issue](https://github.com/fedora-modularity/conu/issues/185).